tests/target/enum.rs

```diff
@@ -27,8 +27,10 @@ enum Bar {
 }
 
 enum LongVariants {
-    First(LOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOONG, // comment
-          VARIANT),
+    First(
+        LOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOONG, // comment
+        VARIANT,
+    ),
```

Comment:
I see, yeah I think that's good, although I'd apply the function inner-width heuristic for tuple structs rather than the struct one (so that example would be a one-liner).